### ISSUE TITLE
Tabs can be removed to simplify OVE

### ISSUE THREAD

**Issue by mfero** [TeselaGen/openVectorEditor#392]
@tnrich Tabbing is a non-intuitive mess. can we remove it or fix it? I'm noticing this is HDE but it should be an OVE issue right? It looks like a demo of draggable tabs, but it does not make sense to have all this flexibility if it is not functional for some purpose. It is just confusing with the multiple interface elements and options. The little dots do not help... to the user it looks like a drag handle, but then is is this strange two element drop down menu. The list of strange behaviors is long and I don't really want to list them all. Lets just radically simplify this asap.

**Comment by tnrich** [comment]
I think I'll need more guidance about how to make this intuitive @mfero. Please do list what you see wrong with the tabs right now and I can try to make them better.

**Comment by gmcarthuriv** [comment]
@tnrich, @mfero,

I'd love to be part of this discussion. Any chance we can set up a video call this week? A discussion would probably be more efficient than going back and forth here.

**Comment by mfero** [comment]
@tnrich @gmcarthuriv sure! Sorry about being obnoxious, and maybe because a few things were buggy... but the tabs just seemed really clunky and a I quickly got the viewer into a state where I could not restore it to the old default view no matter what I tried. We abandoned internal tabs in the rest of the app, but they seem to live on inside OVE. Are we going to bring them back for the rest of the platform as well or are they unique to OVE? I'll schedule a call :)
